test/functional/test_framework/util.py

```diff
@@ -226,7 +226,7 @@ def ceildiv(a, b):
 def get_fee(tx_size, feerate_btc_kvb):
     """Calculate the fee in BTC given a feerate is BTC/kvB. Reflects CFeeRate::GetFee"""
     feerate_sat_kvb = int(feerate_btc_kvb * Decimal(1e8)) # Fee in sat/kvb as an int to avoid float precision errors
-    target_fee_sat = ceildiv(feerate_sat_kvb * tx_size, 1000) # Round calculated fee up to nearest sat
+    target_fee_sat = ceildiv(feerate_sat_kvb * int(tx_size), 1000) # Round calculated fee up to nearest sat
     return satoshi_round(target_fee_sat / Decimal(1e8)) # Truncate BTC result to nearest sat
```

Comment:
In commit "test: Call ceildiv helper with integer" (11d4385819154e637eb34879bf3e4040f85ef194)

Not directly related to this PR, but it seems like this satoshi_round call does nothing and should be dropped. (I suggested this previously https://github.com/bitcoin/bitcoin/pull/22949#discussion_r728064800)

Reply:
Yes, I agree it does nothing since we pass an argument that needs no rounding because `target_fee_sat` is calculated in sat. I removed the call to `satoshi_round` here.

The only difference in behavior I could see is that if we'd call `get_fee` with absurdly high feerates such as
`get_fee(1000, Decimal("1.00000001"))`, we'd get an `decimal.InvalidOperation` assert from `satoshi_round` before because the specified Decimal precision of 8 would not be enough.